### ISSUE TITLE
Hide useless elements for screen readers

### ISSUE THREAD

**Issue by nvdaes** [edrlab/thorium-reader#1014]
Hide elements such as:
- SVG like Import publication, bookmark and other elements which can be available as buttons, for a more consistent and predictable operability and less verbosity with screen readers.
- Hide the toolbar, which cannot be accessed and produces a wrong tree of landmarks with Elements list dialog in NVDA (NVDA+f7).

**Comment by nvdaes** [comment]
Looking at the code, seems that there is a div labelled as toolbar with role region, and this div has even the main div as children. Also, there is a link with role region as the first child of this div... In short, seems that aria labels and roles should be improved. Now this produces too verbosity with NVDA, not always making sense. I think that regions should be restructured.

**Comment by danielweck** [comment]
Hello Noelia, I believe the UI controls accessibility has greatly improved in recent builds, we fixed labelling issues in the toolbar notably. I am closing this issue but of course feel free to re-open to explain what still doesn't work well, if anything <smile> Regards, Daniel

**Comment by nvdaes** [comment]
Thanks Daniel. I"ll open a new issue if I have any suggestion.
